digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -263,12 +262,8 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
   Returns empty result as Json message, not as page not found
   */
   get("/massTransitStopsSafe/:nationalId") {
-    def validateMunicipalityAuthorization(nationalId: Long)(municipalityCode: Int): Unit = {
-      if (!userProvider.getCurrentUser().isAuthorizedToRead(municipalityCode))
-        halt(Unauthorized("User not authorized for mass transit stop " + nationalId))
-    }
     val nationalId = params("nationalId").toLong
-    val massTransitStopReturned = massTransitStopService.getMassTransitStopByNationalIdWithTRWarnings(nationalId, validateMunicipalityAuthorization(nationalId))
+    val massTransitStopReturned = massTransitStopService.getMassTransitStopByNationalIdWithTRWarnings(nationalId)

```

Comment:
why was the userValidation  removed?

Reply:
There are no more restrictions on reading assets